data/vocab/scholar.ttl

```diff
@@ -460,19 +461,24 @@ scholar:mostCitedWork a rdf:Property ;
     rdfs:range lite:Work ;
     rdfs:domain lite:Person .
 
-## Crossref ID properties
+## ID properties
 
 scholar:fainId a rdf:Property ;
     rdfs:label "fain ID"@en ;
     rdfs:comment "A unique identifier for a funding award."@en ;
     rdfs:domain scholar:FundingAward ;
     rdfs:range rdfs:Literal .
+
+scholar:openalexID a rdf:Property ;
```

Comment:
Why does the `fainId` property have the d lower case but here it is capitalized?

Reply:
Good catch it should be openalexId (properties are humpcase according to RDF naming conventions)

Reply:
I pushed an update